### PR TITLE
Use os util on entire of codebase

### PR DESCRIPTION
**Pull request by behnammodi**
we had os util but in some where of code we didn't use it

### CODE REVIEW

File: _util/os.ts [hunk 1]
```diff
@@ -17,4 +19,4 @@ export const osType = (() => {
   return "linux";
 })();
 
-export const isWindows = osType === "windows";
+export const isWindows: boolean = osType === "windows";
```
Comment (kitsonk) [inline _util/os.ts:22]: This is a trivial type annotation and not required.

Comment (behnammodi) [reply _util/os.ts]: Exactly, I will revert this line

**Review by kt3k — APPROVED** [review]
LGTM

**Comment by behnammodi** [comment]
Can I join as member to denoland?

**Comment by behnammodi** [comment]
Can I join as a member in denoland? :)

**Comment by bartlomieju** [comment]
> Can I join as a member in denoland? :)

Sorry, that's not possible, only employees of Denoland have "member" access.

**Pull request merged**